CREATE-NEW-SCALER.md

```diff
@@ -13,7 +13,7 @@ In order to developer a scaler, a developer should do the following:
 
 If you want to deploy locally 
 1. Open the terminal and go to the root of the source code, then build a docker image of KEDA by running `docker build . -t [choose a unique tag for your custom image]`
-2. In the terminal, navigate to the `chart/keda` folder, and run the following command (don't forget to replace the placeholder text in the command) `helm install . --set image.repository=[tag used in step 1],image.pullPolicy=IfNotPresent`.
+2. In the terminal, navigate to the `chart/keda` folder, and run the following command (don't forget to replace the placeholder text in the command) `helm install . --set image.keda=[tag used in step 1],image.pullPolicy=IfNotPresent`.
```

Comment:
Maybe worthy to note that `chart/keda` is now part of the https://github.com/kedacore/charts GitHub repo.

Or add a step to clone the repo before this one

Reply:
yeah, true. I'm going to make that change too.

Reply:
If should be fixed in the last commt!